typings/queries.d.ts

```diff
@@ -13,12 +13,24 @@ export type AllByBoundAttribute = (
   options?: MatcherOptions,
 ) => HTMLElement[]
 
+export type FindAllByBoundAttribute = (
```

Comment:
The `findAllBy` and `findBy` queries accept 4 arguments. The last one is this: https://github.com/kentcdodds/dom-testing-library/blob/96f21a5219332d7dd058f736afcf29758bbdfc11/typings/wait-for-element.d.ts#L3-L7

Reply:
No worries I'll make that adjustment now